charts/fluent-operator/templates/fluentd-fluentd.yaml

```diff
@@ -14,7 +14,7 @@ spec:
         bind: 0.0.0.0
         port: {{ .port }}
   replicas: {{ .replicas }}
-  image: {{ .image.repository }}:{{ .image.tag }}
+  image: {{ template "fluent-operator.image" (tuple .Values.fluentd.image "") }}
```

Comment:
I don't think we want the fluentd image tag to track the fluent-operator version, since these images are released on separate cadences and the versions will not align between fluent-operator and fluentd.

Reply:
See https://github.com/fluent/fluent-operator/pull/1550/files#r2038934721